docs/en/how-to/loading-fixtures.rst

```diff
@@ -106,3 +106,13 @@ then pass ``true`` to the 2nd argument of execute:
 .. code-block:: php
     <?php
     $executor->execute($loader->getFixtures(), append: true);
+
+By default the ``ORMExecutor`` will wrap the purge and the load of fixtures
+in a single transaction, which is the recommended way, but in some cases (for
+example if loading your fixtures is too slow and causes timeouts) you may
+want to wrap the purge and the load of every fixture in its own transaction.
+To do so, you can pass ``false`` to the third argument of ``ORMExecutor``:
```

Comment:
This is a bit odd. You talk about a third argument, but the code uses named arguments. Maybe we should rephrase that.

```suggestion
To do so, construct ``ORMExecutor`` with ``singleTransaction: false``:
```

Reply:
I agree, I did it like this because of the previous sentence, shall I change it too?

> If you want to append the fixtures instead of purging before loading
> then pass ``true`` to the 2nd argument of execute:

Reply:
If you'd like to, yes. 🙂